gp2gp-translator/src/integrationTest/java/uk/nhs/adaptors/pss/translator/EhrExtractHandlingIT.java

```diff
@@ -107,10 +110,36 @@ public void setUp() {
     @Test
     public void handleEhrExtractFromQueue() throws JSONException {
         // process starts with consuming a message from MHS queue
-        sendInboundMessageToQueue("/xml/RCMR_IN030000UK06/payload_part.xml");
+        sendInboundMessageToQueue("/xml/RCMR_IN030000UK06/payload_part.xml", EBXML_PART_PATH);
+
+        // wait until EHR extract is translated to bundle resource and saved to the DB
+        waitAtMost(Duration.ofSeconds(WAITING_TIME)).until(this::isEhrMigrationCompleted);
+
+        // verify generated bundle resource
+        verifyBundle("/json/expectedBundle.json");
+    }
+
+    @Test
+    public void handleEhrExtractWithConfidentialitySchemaFromQueue() throws JSONException {
```

Comment:
This and the test below have very similar names, not something that needs fixing immediately, but each test should be clear about what it's testing.

Reply:
yes, agree, I can take a note of it and do it at the end of all the refactoring